BusyList/Handlers/ReadHandler.cs

```diff
@@ -16,14 +16,8 @@ public void Run(ReadCommand command)
         {
             var task = _taskRepository.GetTaskById(command.Id);
 
-            if (task == null)
-            {
-                Console.WriteLine($"No task found with id {command.Id}");
-            }
-            else
-            {
-                Console.WriteLine($"The following task has been found: {task.Id}: {task.Description}");
-            }
+            if (task == null) return;
```

Comment:
Should let the user know that there was no task with the given ID and THEN return.

Reply:
But won't the second statement be unreachable?  Like this:
 ``` if (task == null) Console.WriteLine($"This task does not exist with the id {task.Id}"); return;
            Console.WriteLine(task.Print());```

Reply:
```csharp
if (task == null)
{
	Console.WriteLine($"No task with id {command.Id} found.");
	return;
}

Console.WriteLine(task.Print());
```

Reply:
Weird last time when I did that it didnt worked properly. I'll try it again then.